client/src/components/common/Card/index.js

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
+// import data from './data';
 import './index.css';
 
-const Card = ({ title, desc }) => {
-  return (
-    <div className="card-box">
-      <div className="card-box__title">
-        <h3 className="title">{title}</h3>
-      </div>
-      <div className="card-box__desc">
-        <p>{desc}</p>
-      </div>
-    </div>
-  );
-};
+export default class Card extends React.Component {
+  state = { data: [] };
 
-export default Card;
+  componentDidMount() {
+    // we will make a request to fetch data by using (Axios)
+  }
+
+  render() {
+    const { data } = this.state;
+    return (
+      <>
```

Comment:
use `<React.Fragment>`

Reply:
what's the difference ??

Reply:
TBH I wasn't familiar with the empty tag and I thought it was a mistake :) With that said, it does look like a mistake and also can be easily overlooked, so I would still go with the fragment, but that's just my personal preference, so up to you..